threedworld-mit/tdw
# Robot joint force

Closes #213 

# How to test:

1. Make sure your version of TDW is up to date (`pip3 install tdw -U` if installed from PyPi or `git pull` if installed from this repo)
1. Download, extract, and run [this build](https://www.dropbox.com/s/1ch80klpj52j9oh/TDW_Linux_robot_joint_force.tar.gz?dl=0)
2. Run [this controller](https://github.com/threedworld-mit/tdw/blob/7b9ee18a1a35e6c47dc6fab6bab37be9493fb66f/Python/example_controllers/robot_torque.py)

# Changelog

### Command API

#### New Commands

| Command                   | Description                        |
| ------------------------- | ---------------------------------- |
| `add_torque_to_revolute`  | Add a torque to a revolute joint.  |
| `add_force_to_prismatic`  | Add a force to a prismatic joint.  |
| `add_torque_to_spherical` | Add a torque to a spherical joint. |

### Example controllers

- Added: `robot_torque.py`
